beast/plotting/plot_param_vs_chi2min.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+"""
+Plot the chi2min versus the fit parametrs
+Used to explore where the bad (and good) fits are located
+"""
+
+from __future__ import print_function
+
+import matplotlib.pyplot as plt
+from astropy.table import Table
+from matplotlib.colors import LogNorm
+from matplotlib import rc
+
+# local imports
+from beastplotlib import (fancify_colname, initialize_parser,
+                          plot_generic, set_params)
+
+def make_param_vs_chi2min_plots(stats, suffix='Exp', figsize=(20,10)):
+    '''Makes a set of 6 diagnostic 2D histograms for BEAST output.
+
+    Parameters
+    ----------
+    stats : astropy Table
+        stats table from BEAST run
+    suffix : str, optional
+        Column type ('Exp', 'Best', 'p16', 'p50', 'p84').
+        Defaults to 'Exp'.
+    figsize : tuple of ints, optional
+        Size of figure to return in inches (width, height).
+        Defaults to (10, 5.5).
+
+    Returns
+    -------
+    fig : matplotlib figure object
+        Figure with diagnostic plots
+    '''
+
+    base_cnames = ['Av', 'logA', 'M_ini', 'Rv', 'f_A', 'Z']
+    cnames = ['{}_{}'.format(n, suffix) for n in base_cnames]
+    ycol = 'chi2min'
+    plot_pairs = [[cnames[0], ycol],
```

Comment:
This is by no means critical, but this line could be simplified to `plot_pairs = [[name, ycol] for name in cnames]`.